### PR TITLE
Feature/add support for action event handlers

### PR DESCRIPTION
**Pull request by RobertByrnes**
Reference issue #28  - Add support for @storybook-actions addon.

### CODE REVIEW

**Comment by RobertByrnes** [comment]
This adds the ability to pass in an array of events to detect if your component is emitting the correct HTML events using the parameters.actions.handles parameter as per @storybook-actions addon. Changes to GenerateStories.php and README.md, there are more possibilities with actions, they are a WIP.
Capture

**Comment by mrtimbrook** [comment]
@RobertByrnes Thanks for adding this! It's looking great! 

One nitpicky thing - Could we remove the `parameters` array and have `actions` as the root array in the `@storybook` directive? There are a lot of things in the `@storybook` directive which gets merged into the story's `parameters` object so I don't think we need to define it in the directive here.

**Comment by RobertByrnes** [comment]
@mrtimbrook Hi, no trouble, this is done and pushed.  I also noticed an issue with serving an asset locally rather than via cdn for something like fontawesome or boxicons.  A cors error is generated for this when the browser tries to access a font in a directory within /public/fonts. etc.   From looking a the storybook docs a workaround to this is to create a file in vendor/area17/blast/.storybook named preview-head.html with cdn links in. This works. Happy to put in PR for this, unless you are doing more with the CORS subject generally...?

File: .storybook/preview.js [hunk 1]
```diff
@@ -38,4 +38,4 @@ export const parameters = {
   }
 };
 
-export const globalTypes = JSON.parse(process.env.STORYBOOK_GLOBAL_TYPES);
+export const globalTypes = JSON.parse(process.env.STORYBOOK_GLOBAL_TYPES);
```
Comment (mrtimbrook) [inline .storybook/preview.js:41]: Can you revert this change? It looks like the new line was removed. I just want to try and keep the history tidy. Thanks!

**Review by mrtimbrook — CHANGES_REQUESTED** [review]
Thanks for updating that! One small change and I'll merge.

If you have a solution for the CORS issues I'd be happy to review a PR.

File: .storybook/preview.js (hunk 1 above)
Comment (RobertByrnes) [reply .storybook/preview.js:41]: @mrtimbrook do you mean the new line at line 40, I've checked and it'd there? or do you mean there is one missing below `export const globalTypes = JSON.parse(process.env.STORYBOOK_GLOBAL_TYPES);`

Comment (RobertByrnes) [reply .storybook/preview.js:41]: @mrtimbrook I'll PR the CORS workaround later on this evening

Comment (mrtimbrook) [reply .storybook/preview.js:41]: @RobertByrnes Yeah, the one at the end of the file (after line 41).
